R/RunWorkflow.R

```diff
@@ -8,6 +8,9 @@
 #'
 #' @param lWorkflow `list` A named list of metadata defining how the workflow should be run.
 #' @param lData `list` A named list of domain-level data frames.
+#' @param lConfig `list` A configuration object with two methods:
+#' - `LoadData`: A function that loads data specified in `lWorkflow$spec`.
+#' - `SaveData`: A function that saves data returned by the last step in `lWorkflow$steps`.
```

Comment:
it might be helpful to add the `4_WorkflowIO.R` example to the examples in this roxygen header, or just make a reference to the example in this param explanation

Reply:
Added in the latest commit.